### ISSUE TITLE
Update SV Mini ATLAS using scaife-viewer/backend

### ISSUE THREAD

**Issue by jacobwegner** [scaife-viewer/sv-mini-atlas#13]

**Comment by jacobwegner** [comment]
This is blocked on the creation of a `scaife-atlas-core`package from [scaife-viewer/backend](https://github.com/scaife-viewer/backend)

**Comment by jacobwegner** [comment]
Deployed to https://mini-stack-a-task-updat-y6iwnd.herokuapp.com/graphql/, but still working on one last pass to optimize / parallelize ingestion.

**Comment by jacobwegner** [comment]
Ingestion is being tracked in another ticket; updating this to show released status.
